package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "topcoat-public",
+  "version": "1.0.0",
+  "description": "",
+  "main": ".eslintrc.js",
+  "scripts": {
+    "lint:js": "npx eslint  --color --ext .js,.json,.ts,.vue .",
+    "lint:js:fix": "npm run lint:js -- --fix",
+    "lint:format": "npx prettier . -c",
+    "lint:format:fix": "npx prettier . --write",
+    "lint:fix": "npm run lint:js:fix && npm run lint:format:fix"
```

Comment:
we could also add stylelint in the future, but I didn't want to add another thing in this PR.